lib/ast-converter.js

```diff
@@ -776,6 +786,13 @@ module.exports = function(ast, extra) {
                     }
                 }
 
+                /**
+                 * Prefix FunctionDeclarations within TypeScript namespaces with "TS"
+                 */
+                if (node.parent && node.parent.kind === SyntaxKind.ModuleBlock) {
+                    functionDeclarationType = "TS" + functionDeclarationType;
```

Comment:
`TSNamespaceFunctionDeclaration`. I'm just wondering if this same syntactic form can be used in other locations or not. (Basically, is "Namespace" too specific for this node or not?)